<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trip</class>
 <widget class="QWidget" name="Trip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="tripOverviewLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>291</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="shoppingButtonsLayout"/>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="tripOverviewLabelLayout">
    <item>
     <widget class="QLabel" name="tripOverviewLabel">
      <property name="text">
       <string>Current Vacation Plan</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="shoppingButtonsLabelLayout">
    <item>
     <widget class="QLabel" name="shoppingButtonsLabel">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>380</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="nextCityButtonLayout">
    <item>
     <widget class="QPushButton" name="nextCityButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Next City</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>171</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="purchaseHistoryLayout">
    <item>
     <widget class="QTableWidget" name="purchaseHistory"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>509</x>
     <y>9</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="purchaseHistoryLabelLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Purchase History</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
